source/gui/settingsDialogs.py

```diff
@@ -554,8 +570,14 @@ def onCategoryChange(self, evt):
 			evt.Skip()
 
 	def onOk(self,evt):
+		for panel in self.catIdToInstanceMap.itervalues():

```

Comment:
I would prefer to see the duplicated code in `onOk` and `onApply` in a separate function, perhaps `_doSave`